whipper/src/main/java/org/whipper/xml/XmlHelper.java

```diff
@@ -203,6 +203,7 @@ public static void loadResult(File f, ExpectedResultHolder erh, Query q) throws
         try(FileInputStream fis = new FileInputStream(f)){
             Result res = (Result)RESULT_UNMARSHALLER.unmarshal(fis);
             QueryResultType qr = res.getQueryResult();
+            qr.setDirectory(f.getParentFile().getParentFile().getName());
```

Comment:
This is method to unmarshall result. Directory will be unmarshalled from the file, no need to set it here.
I believe, we should set this information along with writing error (XmlHelper.writeError(...) method). At least that is my original idea.

Reply:
My mistake, i moved it to writeError method.